.github/workflows/codeforafrica-deploy-prod.yml

```diff
@@ -14,8 +14,10 @@ env:
   DOKKU_REMOTE_URL: "ssh://dokku@ui-1.prod.codeforafrica.org/codeforafrica-ui"
   IMAGE_NAME: "codeforafrica/codeforafrica-ui"
   APP: "codeforafrica"
+  APP_NAME: "codeforafrica-ui"
```

Comment:
Isn't `APP_NAME` supposed to be something visible to the end users i.e. "Code for Africa"? At least that's what the code [here](https://github.com/CodeForAfrica/ui/blob/aef401ce0dd7a137d20584ce892e235f9627152b/apps/codeforafrica/src/utils/site.js#L1) tells me.
Looks like we're using it as something for the backend i.e. Dokku app name... or this APP_NAME is something else completely.